app/grants/templates/grants/shared/landing_qf_active.html

```diff
@@ -0,0 +1,97 @@
+            <a href="https://vitalik.ca/general/2019/12/07/quadratic.html">quadratic funding</a>
+          </span>
+        </h3>
+        <div class="my-3">
+          <a class="btn btn-primary" href="/grants/explore">{% trans 'Explore Grants' %}</a>
+          <a class="btn btn-outline-primary ml-3" href="/grants/new">{% trans 'Create a Grant' %}</a>
+        </div>
+        <div>
+          or
+          <a href="/grants/matching-partners/new">
+            {% trans "become a member of the funder's league" %}
+          </a>
+        </div>
+      </div>
+      <div class="col-auto d-none d-md-block">
+        <img src="{% static 'v2/images/grants/rocket.svg' %}" />
+      </div>
+    </div>
+  </div>
+
+  <!-- This div is where the hero image goes-->
+  <div class="pb-5" id="grant-hero-img">
+    <h1>$500k Funding Pool, Now Live</h1>
+    <div class="text-muted">Sponsored by</div>
+    <div class="my-5">
+      <div class="row">
+        <div class="col-12 col-md-6 order-md-last mb-5 mb-md-0">&nbsp;</div>
+      </div>
+      <div class="m-0 col-12 col-md-6 offset-lg-1 col-lg-5 d-none d-sm-block">&nbsp;</div>
+    </div>
+  </div>
+
+  <!-- Summary of matching pools this round -->
+  <div class="row justify-content-between align-items-center text-secondary mt-3">
+    <div class="col-auto">$125K Community</div>
+    <div class="col-auto">$125K Infra Tech</div>
+    <div class="col-auto">$125K dApp Tech</div>
+    <div class="col-auto">$125K Eask Asia</div>
+    <div class="col-auto">$125K NFTs</div>
+  </div>
+
+  <!-- Ongoing matching pools (CURRENTLY HIDDEN AND NOT IMPLEMENTED) -->
+  {% comment %}
+  <div class="my-5">
+    <div class="row justify-content-start align-items-center">
+      <div class="col-auto"><img height="30rem" src="{% static 'v2/images/ethereum.svg' %}" /></div>
+      <h4 class="col-auto font-weight-bold py-0 my-0 pl-0">Ongoing Matching Pools</h4>
+    </div>
+  </div>
+  {% endcomment %}
+
+  <!-- Collections -->
+  <div class="my-5">
+    <div class="row justify-content-between align-items-end">
+      <div class="col-auto">
+        <div class="row justify-content-start align-items-center">
+          <img
+            class="col-auto"
+            height="30rem"
+            src="{% static 'v2/images/grants/collections-icon.svg' %}"
+          />
+          <div class="col-auto">
+            <h4 class="font-weight-bold py-0 my-0 pl-0">Collections</h4>
+            <div class="text-muted d-none d-md-block">
+              Selections of grants curated by the community
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col-auto text-muted d-block d-md-none">
+        Selections of grants curated by the community
+      </div>
+      <div class="col-auto">
+        <a href="/grants/collections">{% trans "View all collections" %}</a>
+        <i class="ml-2 text-primary fas fa-long-arrow-alt-right"></i>
+      </div>
+    </div>
+    <!-- TODO show collections-->
```

Comment:
this mean this pr is ready or still need work?

Reply:
This TODO was one of the things that became a time suck so I postponed it for now with that TODO comment

So to answer your question—up to you. Since we're merging things into `grants-9` now, my preference would to "merge fast" even if incomplete. That way the `grants-9` branch always has the latest completed state of the grants redesign, and it  minimizes merge/rebase conflicts